sdk/search/search-documents/samples-dev/synonymMapOperations.ts

```diff
@@ -20,7 +20,12 @@ async function createSynonymMap(synonymMapName: string, client: SearchIndexClien
     name: synonymMapName,
     synonyms: ["United States, United States of America => USA", "Washington, Wash. => WA"],
   };
-  await client.createSynonymMap(sm);
+  await client.createSynonymMap(sm, {
```

Comment:
Can you add a trivial test that asserts `_response` is defined on the result of some operation?

Reply:
Done. Added a Unit test on core-http-compat package